Components/login.css

```diff
@@ -1,14 +1,15 @@
 .login-container{
     background-color: var(--color-primary);
     width: auto;
```

Comment:
width is by default Auto try to avoid using that

Reply:
width is inherited here, I am overwriting that value with 0; but thanks for pointing it out; i was not aware of width default value...😁